.gitignore

```diff
@@ -0,0 +1 @@
+.terraform/
```

Comment:
A gitignore was added to ignore the various terraform provider/module information. It is generally recommended to ignore this file.